analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -393,7 +393,7 @@ class Pip(
         releaseNode ?: return RemoteArtifact.EMPTY
 
         // Prefer python wheels and fall back to the first entry (probably a sdist).
-        val binaryArtifact = releaseNode.asSequence().find {
+        val binaryArtifact = releaseNode.find {
```

Comment:
This I also saw in the inspections, but could not comprehend why the conversion to a sequence would be redundant. Can you? (And explain in the commit message.)

Reply:
I believe I got it, see commit message.